samples/highcharts/website/xmas-2021/demo.js

```diff
@@ -0,0 +1,3937 @@
+                        type: 'Point',
+                        coordinates: [-118.24, 34.05]
+                    },
+                    dataLabels: {
+                        enabled: false
+                    },
+                    states: {
+                        inactive: {
+                            enabled: false
+                        }
+                    }
+                }],
+                color: 'transparent'
+            }, false);
+            chart.redraw(false);
+        }
+    };
+
+    const starData = [];
+    const getStarData = function () {
+        for (let ii = 0; ii <= 40; ++ii) {
+            const newX = randomIntFromInterval(0, 10.5);
+            const newY = randomIntFromInterval(5, 8);
+            starData.push([newX, newY]);
+        }
+
+    };
+    getStarData();
+
+    Highcharts.getJSON(
+        'https://cdn.jsdelivr.net/gh/highcharts/highcharts@2e11000c966a20f08afc4e0927b91df99821de99/samples/data/world-countries.topo.json',
+        topology => {
+
+            // Convert the topoJSON feature into geoJSON
+            const geojson = window.topojson.feature(
+                topology,
+                // For this demo, get the first of the named objects
+                topology.objects[Object.keys(topology.objects)[0]]
+            );
+            geojson.copyrightUrl = topology.copyrightUrl;
+            geojson.copyrightShort = topology.copyrightShort;
+
+            chart = Highcharts.mapChart('ball-chart', {
+                chart: {
+                    map: geojson,
+                    margin: 0,
+                    backgroundColor: 'url(#bg-gradient)',
+                    events: {
+                        load: function () {
+                            const chart = this;
+                            let count = 0;
+
+                            chart.series[3].points[0].setState('hover');
+
+                            if (!big) {
+                                chart.update({
+                                    plotOptions: {
+                                        pie: {
+                                            center: [70, 5]
+
+                                        }
+                                    }
+                                });
+                            }
+
+
+                            //spin
+                            setTimeout(function () {
+                                setInterval(function () {
+                                    if (count < 360) {
+                                        chart.update({
+                                            mapView: {
+                                                projection: {
+                                                    name: 'Orthographic',
+                                                    rotation: [count, -30]
+                                                }
+                                            }
+                                        });
```

Comment:
It loooks like this `chart.update` is called with animation. Fourth parameter, true by default unless `chart.animation` is set in the chart options. When updating the chart in a fast loop like this, turning off animation is important so that the browser doesn't have to compute redundant frames between the updates.